app/controllers/follow_requests_controller.rb

```diff
@@ -26,7 +27,8 @@ def create
 
     respond_to do |format|
       if @follow_request.save
-        format.html { redirect_back fallback_location: root_url, notice: "Follow request was successfully created." }
+        # format.html { redirect_back fallback_location: root_url, notice: "Follow request was successfully created." }
```

Comment:
You can delete/uncomment this one now

Reply:
I would actually only uncomment it and leave it there. That way, if your user's browser is blocking JS the HTML response will take over and your website won't break.

Reply:
I also prefer this way. Uncomment and keep it there